jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/AbstractAnnotator.java

```diff
@@ -95,4 +95,8 @@ public GenerationConfig getGenerationConfig() {
         return generationConfig;
     }
 
+	@Override
```

Comment:
Hmm... I used in tabs in files where there were tabs and spaces if there were spaces. I'll verify this and fix it if needed.